.github/workflows/release-test-version.yml

```diff
@@ -19,7 +19,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [macos-latest, macos-13, windows-latest, ubuntu-latest]
+        include:
+          - platform: macos-latest
+            app-path: "out/k6 Studio-darwin-arm64/k6 Studio.app/Contents/MacOS/k6-studio"
+          - platform: macos-13
+            app-path: "out/k6 Studio-darwin-x64/k6 Studio.app/Contents/MacOS/k6-studio"
+          - platform: windows-latest
+            app-path: "out\\k6 Studio-win32-x64\\k6-studio.exe"
+          - platform: ubuntu-latest
+            app-path: "out/k6 Studio-linux-x64/k6-studio"
```

Comment:
`app-path` is a custom argument received by the Webdriver script to specify where the binary lives for each OS.